src/index.ts

```diff
@@ -47,20 +51,26 @@ export async function startNotarize(opts: NotarizeStartOptions): Promise<Notariz
     }
     d('zip succeeded, attempting to upload to apple');
 
+    const notarizeStartOpts = [
+      'altool',
+      '--notarize-app',
+      '-f',
+      zipPath,
+      '--primary-bundle-id',
+      opts.appBundleId,
+      '-u',
+      makeSecret(opts.appleId),
+      '-p',
+      makeSecret(opts.appleIdPassword),
+    ];
+
+    if (opts.ascProvider !== '') {
```

Comment:
This should just be a truthy check

```suggestion
    if (opts.ascProvider) {
```